app/com/linkedin/drelephant/tuning/TuningHelper.java

```diff
@@ -122,4 +133,52 @@ public static void updateJobExecution(JobExecution jobExecution) {
       logger.error("Exception updating job execution " + jobExecution.id, e);
     }
   }
+
+  public static Double getContainerSize(Double memory) {
+    return Math.ceil(memory / ONE_GB) * ONE_GB;
+  }
+
+  public static Double getHeapSize(Double heapSize) {
+    if (heapSize < MINIMUM_HEAP_SIZE_MB) {
+      heapSize = MINIMUM_HEAP_SIZE_MB * 1.0;
+    }
+    return heapSize;
+  }
+
+  public static Double parseMaxHeapSizeInMB(String data) {
+    try {
+      logger.debug(" Heap data " + data);
+      Pattern pattern = Pattern.compile(PATTERN_FOR_MEMORY);
+      Matcher m = pattern.matcher(data);
+      String maxHeapSize;
+      if (m.find()) {
+        maxHeapSize = m.group().trim();
+        maxHeapSize = maxHeapSize.substring(4);
+        return MemoryFormatUtils.stringToBytes(maxHeapSize)/(1024.0*1024.0);
+      }
+    } catch (Exception e) {
+      logger.error(" Error gettting max heap ", e);
+    }
+
+    logger.info(" Max heap size not found ");
+    return DEFAULT_MAX_HEAP_SIZE * 1.0;
+  }
+
+  public static double getTimeInMinute(String value) {
```

Comment:
Kindly add Unit Test for this Helper function.

Reply:
yup forgot to write test case for the same . Now written

Reply:
Resolved